src/app/qtlocalpeer/qtlocalpeer.cpp

```diff
@@ -96,41 +92,14 @@ namespace QtLP_Private
 
 const char* QtLocalPeer::ack = "ack";
 
-QtLocalPeer::QtLocalPeer(QObject *parent, const QString &appId)
+QtLocalPeer::QtLocalPeer(const QString &path, QObject *parent)
```

Comment:
Since it has now ceased to be compatible with the original QtLocalPeer class, I would finally integrate it into our codebase by applying the coding style, naming and placement of classes/files.

P.S. I don't insist it to be done in this PR if you don't want to deal with it.

Reply:
>Since it has now ceased to be compatible with the original QtLocalPeer class, I would finally integrate it into our codebase by applying the coding style, naming and placement of classes/files.

Agree.

>P.S. I don't insist it to be done in this PR if you don't want to deal with it.

Let's leave it for another PR, I'm not motivated to handle it.